spec/app_spec.rb

```diff
@@ -219,4 +219,27 @@
     expect(response['current_page']).to eq 2
   end
 
+  it "should return a list of addresses that have been updated since a certain date" do
+    Timecop.freeze(DateTime.new(2013,1,1))
+
+    5.times do
+      FactoryGirl.create(:address)
+    end
+
+    Timecop.return
+
+    date = DateTime.now
+
+    5.times do
+      FactoryGirl.create(:address)
+    end
+
+    get 'addresses', { updated_since: date.to_s }
```

Comment:
Still not sure this is precise enough; don't we want to support ISO8601 only, presumably? It seems vague, but perhaps it doesn't matter.